.cloudbuild/source-checks.yml

```diff
@@ -1,6 +1,6 @@
 steps:
   - id: gokart
-    name: golang:1.16
+    name: golang:1.19
```

Comment:
Should this be `23`?

Reply:
gokart doesnt run on 1.23, this is the most up-to-date image it can run on (luckily its compatible with the rest of the changes)